google-cloud-bigquery/README.md

```diff
@@ -23,16 +23,16 @@ If you are using Maven, add this to your pom.xml file
 <dependency>
   <groupId>com.google.cloud</groupId>
   <artifactId>google-cloud-bigquery</artifactId>
-  <version>0.37.0-beta</version>
+  <version>1.19.0</version>
 </dependency>
 ```
 If you are using Gradle, add this to your dependencies
 ```Groovy
-compile 'com.google.cloud:google-cloud-bigquery:0.37.0-beta'
+compile 'com.google.cloud:google-cloud-bigquery:1.19.0'
 ```
 If you are using SBT, add this to your dependencies
 ```Scala
-libraryDependencies += "com.google.cloud" % "google-cloud-bigquery" % "0.37.0-beta"
+libraryDependencies += "com.google.cloud" % "google-cloud-bigquery" % "1.19.0"
```

Comment:
Let's do this in two stages so that people don't have broken instructions in READMEs.

- In this commit, revert the updates to the README here.
- In the next release, the versions here will get automatically corrected from versions.txt
- After the next release, update the root README to move BigQuery to the GA section, and in this file, remove the verbiage about the client being a work in progress.

Reply:
Makes sense!